packages/terra-clinical-label-value-view/src/LabelValueView.jsx

```diff
@@ -44,13 +50,32 @@ const LabelValueView = ({
   } else if (textValue) {
     textValueSection = <div className={cx('value')}>{textValue}</div>;
   }
+  const termDefinition = (
+    <React.Fragment>
+      <dt className={cx('label')}>{label}</dt>
+      <dd className={cx('value-wrapper')}>
+        {textValueSection}
+        {children}
+      </dd>
+    </React.Fragment>
+  );
+
+  /**
+   * If the LabelValueView is included by the consummer into a description list, then we just return the term and the definition.
+   * Otherwise, we are sending the whole description list for the consumer to use.
```

Comment:
Can we reword to:
```suggestion
   * If LabelValueView is a child of a description list (<dl>), then just the term and definition are returned.
   * Otherwise, a description list encapsulating the term and definition is returned.
   * The reason for this is that <dt> and <dd> tags must always be children of a <dl> to be valid html.
```